README.md

```diff
@@ -20,6 +20,7 @@ Example servers running Nukkit
 - **play.EaseCation.net**
 - **play.GameTeam.cz**
 - **MultiLabs.net**
+- **play.2p2e.net**
```

Comment:
I was thinking of just removing this from the README altogether otherwise people will constantly ask for theirs to be on it.